### DISCUSSION TITLE
creating a room

### DISCUSSION THREAD

**Discussion by inst5** [cables-gl/cables_docs#399] [Answers & Questions] [answered]
Hi,
curious to develop a VR experience with cables. 
One very basic questio I have. How can I create my own enironment / room?
So, like the basic physics room in cables.
Are there any special tools you would suggest?

thanks fo feedback!

**Comment by steam0r** [comment, marked as answer]
there is a template for a room in cables. firing up blender and working along these lines should get you going:

![Bildschirmfoto 2022-11-04 um 09 52 27](https://user-images.githubusercontent.com/151271/199931826-8bd1b321-2f9a-47c2-ad55-d2adad3f3786.png)

you can also always look around public or example patches and clone them to start from there.

**Reply by inst5** [reply]
HI,
a late Thank you for this.  Do you know other tools than blender to create virtual rooms quicker/easier?
thanks!
